<ui version="4.0" >
 <class>configDialog</class>
 <widget class="QWidget" name="configDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Enter here the path for mplayer executable file.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="kcfg_mplayerbin" />
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Custom arguments for mplayer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="kcfg_customargs" />
   </item>
   <item>
    <widget class="KEditListBox" name="kcfg_noextensions" >
     <property name="title" >
      <string>Blacklisted File Extensions</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
